webapp-django/crashstats/symbols/views.py

```diff
@@ -103,7 +103,7 @@ def unpack_and_upload(iterator, symbols_upload, bucket_name, bucket_location):
         key = bucket.get_key(key_name)
 
         # let's assume first that we need to add a new key
-        prefix = "+"
+        prefix = '+'
```

Comment:
This change is unrelated but it was burning my retinas when I say it was the only string not in single quotes. Call me anal :)